src/index.spec.ts

```diff
@@ -819,5 +819,85 @@ describe("ServerlessPlugin", () => {
       // The service and env tags will be set with the values given in the provider instead
       expect(function_).toHaveProperty("tags", {});
     });
+
+    it("Does not attempt add execution log groups if execution logging env variable is false", async () => {
```

Comment:
Technically the plugin's configuration settings aren't environment variables. Environment variables are variables you set outside of a program in your computer's operating system, that the operating system then gives the program access to. For example, when you run:

```
DD_API_KEY=xxx sls deploy
```

DD_API_KEY is an environment variable, and the Serverless Framework is given access to that variable by the operating system.

Environment variables are in the format of KEY=value, they don't usually support nesting/JSON/YML (although you can shove JSON into the value if you want to).

What we have for configuring our plugin is just a configuration YML file that we read, they aren't provided to us by the operating system. The settings aren't environment variables per se. So I would change the references from "environment variables" to just "settings" or "configuration options".

Reply:
Also, nice tests :+1:

Reply:
This is actually testing the presence of the env variable in `index.ts` at line 136.

Reply:
Right, but unless I'm missing something `subscribeToExecutionLogs` is not an env variable... happy to discuss this one more on Zoom if you want